pkg/controller/directvolumemigration/task.go

```diff
@@ -420,11 +420,11 @@ func (t *Task) Run() error {
 			return liberr.Wrap(err)
 		}
 	case WaitForRsyncClientPodsCompleted:
-		completed, failed, err := t.haveRsyncClientPodsCompletedOrFailed()
+		completed, failed, pending, err := t.haveRsyncClientPodsCompletedFailedOrPending()
 		if err != nil {
 			return liberr.Wrap(err)
 		}
-		if completed {
+		if completed && !pending {
```

Comment:
why do you need pending here? if the pods are in a pending state doesn't that automatically mean they are not completed?

Reply:
Yip You are on the right track. removed the additional variable since it is not neeed